README.md

```diff
@@ -39,6 +39,89 @@ bundle install
 rake -T
 ```
 
+## Dockerfile
+
+### Build image
+```
+docker build -t <image_name> .
+```
+### Run tests in Docker
+
+Driver which is used by default is **headless chrome**
+
+Use **docker run** command to create and run container.
+
+- In order to run container by default:
+```
+docker run -d --name <container_name> <image_name>
+```
+
+- To run container using firefox headless browser need to override environment variable:
+**SEXY_SETTINGS**
+```
+docker run -e "SEXY_SETTINGS=driver=headless_firefox" -d --name <container_name> <image_name>
+```  
+
+- Connect to created container using this command:
+```
+docker exec -it <container_name/ID> /bin/bash
+```
+
+- Run tests with the next command using container's terminal:
+```
+bundle exec rake
+```
+
+### Start and Stop containers
+- Use docker commands such as **docker start** and **docker stop** to restart container or stop container:
+```
+docker start <container_name/ID>
+docker stop <container_name/ID>
+```
+
+## Docker compose
+
+### Override default driver
```

Comment:
I am not sure we need this section, the reason is specifying the env inside of the container